### PR TITLE
Messaging template fixes

### PR DESCRIPTION
**Pull request by Thurfjell**
### Ticket:
Fix templating to not require lang in query, in the rest api / sdk plus some smaller fixes.
Added a new ui for templates
- [15875](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_backlogs/backlog/Life%20Events/Stories?workitem=15875)

### Description

Please include a summary of the changes and the related issue.

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [ ] **New feature**
- [ ] **Dev change**

### Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works

### Screenshots:

N/A

### CODE REVIEW

File: apps/messaging/tsconfig.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
+    "target": "ES6",
```
Comment (edge33) [inline apps/messaging/tsconfig.json:15]: curiosity: why is this change necessary?

**Comment by edge33** [comment]
LGTM

File: apps/messaging/tsconfig.json (hunk 1 above)
Comment (Thurfjell) [reply apps/messaging/tsconfig.json:15]: To use nice things like new Set([...]) eg.

**Review by GiuliaTeggi — APPROVED** [review]
